static/_redirects

```diff
@@ -18,7 +18,7 @@
 /ru/docs/     /ru/docs/home/ 301!
 /uk/docs/     /uk/docs/home/ 301!
 /vi/docs/     /vi/docs/home/ 301!
-/zh/docs/     /zh/docs/home/ 301!
+/zh/docs/     /zh-cn/docs/home/ 301!
```

Comment:
(nit)
The `/zh-cn/` redirect is perhaps more important.
```suggestion
/zh/docs/     /zh-cn/docs/home/ 301!
/zh-cn/docs/     /zh-cn/docs/home/ 301!
```

Reply:
done.